src/cfnlint/formatters/__init__.py

```diff
@@ -167,7 +167,7 @@ def print_matches(self, matches, rules=None, filenames=None):
         del rules
 
         return json.dumps(
-            matches, indent=4, cls=self.CustomEncoder,
+            matches, indent=1, cls=self.CustomEncoder,
```

Comment:
Is this one necessary?  Seems unrelated to the issue

Reply:
Agreed, not necessary and unrelated, just included it in the sweep for no strong reason 🤷 Feel free to keep or remove this one